content/docs/instrumenting/exporters.md

```diff
@@ -83,3 +82,4 @@ practices.
    * Clojure: [prometheus-clj](https://github.com/soundcloud/prometheus-clj)
    * Go: [go-metrics instrumentation library](https://github.com/armon/go-metrics)
    * Go: [gokit](https://github.com/peterbourgon/gokit)
+   * [Hystrix metrics publisher](https://github.com/soundcloud/prometheus-hystrix)
```

Comment:
I was unsure on that, it's more JVM than Java per-se due to how the JVM works.

Let's do Java/JVM.